packages/web-config-server/src/apiV1/index.js

```diff
@@ -26,6 +32,7 @@ export const getRoutesForApiV1 = () => {
   api.get('/getUser', catchAsyncErrors(getUser()));
   api.post('/login/oneTimeLogin', catchAsyncErrors(oneTimeLogin));
   api.post('/signup', catchAsyncErrors(appSignup()));
+  api.post('/resetPassword', catchAsyncErrors(appRequestResetPassword()));
```

Comment:
So weird that this route even worked before?!

Reply:
To be honest, I hadn’t tested it. At the time I was just trusting the unit tests would catch regressions

I think I’ll need to do some serious manual testing before merging this one